test/test_jit.py

```diff
@@ -1178,7 +1178,7 @@ def observe(x, name):
 
         m = torch.jit.script(fn)
         # Insert observers
-        torch._C._jit_pass_insert_observers(m.graph, observe)
+        torch._C._jit_pass_insert_observers(m, observe)
```

Comment:
Is this just for testing? This pass relies on Values's unique name, which is not guaranteed to be stable @ZolotukhinM @nishantpdce

Reply:
@eellison, this pass doesn't rely on the names. It records the names, but only for better debugability. It would work if we use any unique id, and it doesn't have to be the name (but using name allows us to more conveniently examine the data when the names are available).

If we "break" the names at any point, we should just remove the corresponding check from the test - please let me know if you think we need to do it now. The pass itself would not break from that.

Reply:
Yes, we should remove all calls to uniqueName() as something that affects output. I made unrelated changes and the quantization tests broke as a result

Reply:
No problem, I'll remove it from the test.

Reply:
Here is the PR removing these checks: #21657